.circleci/config.yml

```diff
@@ -109,8 +110,11 @@ workflows:
       - prettier: {requires: [cache-yarn-linux]}
       - eslint: {requires: [cache-yarn-linux]}
       - data: {requires: [cache-yarn-linux]}
-      - ios-nightly: {requires: [danger, flow, jest, prettier, eslint, data]}
-      - android-nightly: {requires: [danger, flow, jest, prettier, eslint, data, cache-bundler-linux]}
+      - ios-nightly:
+          requires: [danger, flow, jest, prettier, eslint, data]
```

Comment:
Good bot, but I want the above lines to be the same style. Just because it exceeds some magic line doesn't mean we should change our style. I like this better, though; it's more readable.